connect/runtime/src/test/java/org/apache/kafka/connect/runtime/SourceTaskOffsetCommitterTest.java

```diff
@@ -71,122 +74,74 @@ public void setup() {
                 Long.toString(DEFAULT_OFFSET_COMMIT_INTERVAL_MS));
         WorkerConfig config = new StandaloneConfig(workerProps);
         committer = new SourceTaskOffsetCommitter(config, executor, committers);
-        Whitebox.setInternalState(SourceTaskOffsetCommitter.class, "log", mockLog);
```

Comment:
I did not manage to find such a functionality in Mockito, so I decided to get rid of it.